Resources/Prototypes/Catalog/Fills/Crates/engines.yml

```
@@ -120,12 +120,14 @@
   id: CrateEngineeringSolar
   parent: CrateEngineering
   name: solar assembly crate
-  description: Parts for constructing solar panels and trackers.
+  description: A kit with solar flatpacks and glass to construct ten solar panels and trackers.
```

```suggestion
  description: A kit with solar flatpacks and glass to construct ten solar panels.
```
Tracker is not included, it needs special electronics to build (even though it currently does literally nothing).